content/en/policies/best-practices/check_deprecated_apis.md

```diff
@@ -0,0 +1,50 @@
+---
+title: "Check Deprecated Apis"
+category: Best Practices
+version: 
+subject: 
```

Comment:
We need a subject for this so it'll be accessible to a filter, and I'm not sure it's best to list all of these resources by name in a comma-separated list. Ideas?

Reply:
How about using `Kubernetes API` as the subject?

I'll also fix the title...

Reply:
I was kind of thinking something along those lines, so I think that'll work. Hopefully the filter logic doesn't break when it encounters a space (not sure that was tested).